doc/source/parameters/cfd/boundary_conditions_multiphysics.rst

```diff
@@ -122,11 +122,49 @@ For VOF boundary conditions (multiphase flow), the possible ``types`` are ``none
 
 * ``id`` is the number associated with the boundary condition. By default, Lethe assumes that the id is equivalent to the number of the bc.
 
-* ``type``: This is the type of boundary condition been imposed. At the moment, choices are:
+* ``type``: This is the type of boundary condition being imposed. At the moment, choices are:
     * ``none`` for which nothing happens.
     * ``peeling/wetting`` for the fluid can attach to (`wet`) or detach from (`peel`) the boundary. The parameters for peeling/wetting are defined in the :doc:`volume_of_fluid` subsection of the parameter file.
     * ``dirichlet`` for inlet and outlet boundary conditions, to specify which fluid should be at the selected boundary.
 
 .. note::
     For periodic boundary conditions, there is no need to specify anything in the ``boundary conditions VOF`` subsection. The periodic boundary condition must be specified in the ``boundary conditions`` subsection (see :doc:`boundary_conditions_cfd`).
+    
+    
+Cahn-Hilliard
+^^^^^^^^^^^^^^
+
+For Cahn-Hilliard boundary conditions, the possible ``types`` are ``noflux`` (default), ``dirichlet`` and ``angle_of_contact``. The default parameters for ``dirichlet`` and ``angle_of_contact`` are shown:
+
+.. code-block:: text
+
+    subsection boundary conditions cahn hilliard
+    set number                  = 2
+    set time dependent          = false
+        subsection bc 0
+            set id 		= 0
+            set type            = dirichlet
+            subsection phi
+                set Function expression = 0
+            end
+         subsection bc 1 
+            set id              = 1
+            set type            = angle_of_contact
+            set angle value     = 90 # The angle is given in degrees (°) 
+         end
+    end
+    
+.. warning::
+   The ``number`` of boundary conditions must be specified explicitly. This is often a source of error.
```

Comment:
The warning is already mentioned at many places. No need to  say it again here I think :).